extensions/interactions/LogicProof/static/js/completeSpec.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Complete tests for LogicProof interaction JS.
  */
 
-errorWrapper = function(
+var errorWrapper = function(
```

Comment:
> > Should we move this into the describe block if it's not meant to be in global scope? Ditto for sharedErrorWrapper.
>
> Ah no, this needs to be in global scope. This was producing errors since the declaration for errorWrapper was different in different tests. I first moved it to globals but renaming it was a better option. This is fixed now.

But why does it need to be in global scope? I thought we did not want to pollute the global scope -- can we find a way to take it out of global scope?

Reply:
Fixed - moved into describe block.